### PR TITLE
Scaffolding base spk ring commands

### LINKED ISSUE #1067: Stub SPK rings commands and add starting documentation
**Acceptance Criteria:**
- [ ] Scaffold out `spk rings` commands (add, delete, set-default)
- [ ] Create `.md` files with documentation with sample use cases.
- [ ] Add unit tests to exercise inputs.

**Additional context:**
This is the first step towards ring implementation: https://github.com/microsoft/bedrock/issues/955

**Does this require updates to documentation?:**
yes

### PR DESCRIPTION
**Pull request by mtarng**
Closes: https://github.com/microsoft/bedrock/issues/1067

No real logic in `spk ring` commands.

### CODE REVIEW

File: src/commands/ring/delete.test.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+describe("test valid function", () => {
+  it("negative test", async () => {
+    try {
+      const tmpDir = createBedrockYaml();
+      checkDependencies(tmpDir);
+      expect(true).toBe(false);
+    } catch (e) {
+      expect(e).not.toBeNull();
+    }
+  });
```
Comment (dennisseah) [inline src/commands/ring/delete.test.ts:24]: nit: maybe this way is better
```
      expect(checkDependencies(createBedrockYaml())).toThrow();
```

File: src/commands/ring/add.decorator.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "command": "add <ring-name>",
+  "alias": "a",
+  "description": "Add a new ring into the current working directory project repository. This will affect all services within the project repository."
```
Comment (dennisseah) [inline src/commands/ring/add.decorator.json]: add
```
   "disabled": true
```
to disable doc generation

**Review by dennisseah — APPROVED** [review]

File: src/index.ts [hunk 3]
```diff
@@ -24,6 +24,7 @@ const commandModules = [
   "hld",
   "infra",
   "project",
+  "ring",
```
Comment (andrebriggs) [inline src/index.ts]: LGTM! Only question is if we should expose the rings commands right now.

**Review by andrebriggs — APPROVED** [review]

File: src/index.ts (hunk 3 above)
Comment (dennisseah) [reply src/index.ts]: comment out this liner until the commands are ready :-)
uncomment it during development. and unit tests will continue will run regardless :-)

**Review by evanlouie — APPROVED** [review]

**Review by yradsmikham — APPROVED** [review]

**Pull request merged**